fetch-crypto-assets.js

```diff
@@ -0,0 +1,65 @@
+const file = "./data.json"
+
+async function getJSON(jsonFile) {
```

Comment:
What do you think about renaming the param to simply `url`? `jsonFile` kinda implies that you're providing a json file as a paramter (not true)